stdlib/2and3/socket.pyi

```diff
@@ -590,7 +590,7 @@ def create_connection(address: Tuple[Optional[str], int],
 def getaddrinfo(
         host: Optional[Union[bytearray, bytes, Text]], port: Union[str, int, None], family: int = ...,
         socktype: int = ..., proto: int = ...,
-        flags: int = ...) -> List[Tuple[int, int, int, str, Tuple[Any, ...]]]:
+        flags: int = ...) -> List[Tuple[AddressFamily, int, int, str, Tuple[Any, ...]]]:
```

Comment:
This should be within an `if sys.version >= (3, 4):` block, because on 2.7 the first tuple member is still an int.

Also, it looks like the second tuple member is a SocketKind enum, at least on 3.6.

Reply:
There's already https://github.com/python/typeshed/blob/master/stdlib/2and3/socket.pyi#L454 which handles the < 3.4 case by defining `AddressFamily` to be int.

Good point about `SocketKind`, I'll add that and re-push. I was focused too much on the `AddressFamily` to remember the next argument...